tests/app/shared/issue-tables/issue-tables.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { IssueTablesComponent } from '../../../../src/app/shared/issue-tables/issue-tables.component';
+
+describe('IssueTablesComponent', () => {
+    let issueTablesComponent: IssueTablesComponent;
+
+    describe('fitTitleText()', () => {
```

Comment:
Since it is an instance method, `.fixTitleText()` should be a better name for the description instead!

Reply:
Alright!